databricks_cli/sdk/api_client.py

```diff
@@ -111,7 +111,8 @@ def perform_query(self, method, path, data = {}, headers = None):
         with warnings.catch_warnings():
             warnings.simplefilter("ignore", exceptions.InsecureRequestWarning)
             if method == 'GET':
-                resp = self.session.request(method, self.url + path, params = data,
+                translated_data = {k: _translate_boolean_to_query_param(data[k]) for k in data}
```

Comment:
I personally prefer to do `{k: f(v) for k,v in data.items()}` but the intent is clear and the perf difference is negligible here.